src/battle-dex.ts

```diff
@@ -443,29 +443,27 @@ const Dex = new class implements ModdedDex {
 		return template;
 	}
 
-	/** @deprecated */
```

Comment:
This is still deprecated. :[ The Preact client doesn't use this.

Reply:
oh my bad, i didnt mean to remove it